tracee-ebpf/main.go

```diff
@@ -1118,37 +1126,6 @@ func checkRequiredCapabilities() error {
 	return nil
 }
 
-func missingKernelConfigOptions() []string {
-
-	requiredConfigOptions := map[uint32]string{
-		helpers.CONFIG_BPF:           "y",
-		helpers.CONFIG_BPF_SYSCALL:   "y",
-		helpers.CONFIG_KPROBE_EVENTS: "y",
-		helpers.CONFIG_BPF_EVENTS:    "y",
-	}
-
-	kConfig := helpers.KernelConfig{}
-	err := kConfig.InitKernelConfig()
-	if err != nil {
-		// we were not able to get kernel config - ignore the check and try to continue
```

Comment:
This check was made to solve the problem I mentioned above.
Also see https://github.com/aquasecurity/tracee/issues/745, https://github.com/aquasecurity/tracee/pull/746